wcomponents-core/src/main/java/com/github/bordertech/wcomponents/AbstractWComponent.java

```diff
@@ -1529,8 +1420,6 @@ public String getTag() {
 	}
 
 	/**
-	 * {@inheritDoc}
```

Comment:
Please retain this doclet as the inheritance needs to be explicit due to the other comments in the block.